charts/karmada/values.yaml

```diff
@@ -21,7 +21,8 @@ installMode: "host"
 clusterDomain: "cluster.local"
 
 ## @param components component list
-components: []
+components:
```

Comment:
I wonder if these format changes automatically discovered by the script or how are they identified.

Reply:
The change is caused by vscode plugin `vscode-helm`.  It is an automatic formatting tool